descriptions/Engine.Xors3D.md

```diff
@@ -0,0 +1,7 @@
+Xors3d is a middleware solution for game developers, born as a library which extends functionality of Blitz3d by implementing a DirectX9-rendering.
```

Comment:
https://web.archive.org/web/20131208100633/http://xors3d.com/xors3d/overview
It's a good enough link